index.d.ts

```diff
@@ -18,6 +18,20 @@ declare class Emittery {
 	*/
 	static mixin(emitteryPropertyName: string, methodNames?: readonly string[]): Function;
 
+	/**
+	Fires when an event listener was added.
+	
+	An object with `listener` and `eventName` (if `on` or `off` was used) is provided.
+	*/
+	static readonly listenerAdded: unique symbol;
```

Comment:
Can you add a usage example to this and `listenerRemoved`? Readme too.

Reply:
Fixed!